source/base/data_out_base.cc

```diff
@@ -7895,7 +7895,7 @@ DataOutInterface<dim, spacedim>::create_xdmf_entry(
   const double                      cur_time,
   const MPI_Comm &                  comm) const
 {
-  unsigned int local_node_cell_count[2], global_node_cell_count[2];
+  std::uint64_t local_node_cell_count[2], global_node_cell_count[2];
```

Comment:
You also have to fix the data type of the MPI call a couple of lines down from here.

Reply:
Thank you, good catch.  Should be fixed now!